<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mingdan</class>
 <widget class="QDialog" name="Mingdan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>点名</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Mingdan { 
border-image: url(:/image/666.jpg); 
} 
#Mingdan * { 
border-image:url(/image/666.jpg); 
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;</string>
   </property>
   <property name="text">
    <string>点击学生行会有语音点名，对未到的取消签到框，可编辑加减分项，保存按钮可以保存成文本文件方便记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/toum.png);
font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;
border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>保存到桌面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;
image: url(:/image/toum.png);
border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>保存在云端</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>132</y>
     <width>1061</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QTableWidget" name="tableWidget">
      <property name="minimumSize">
       <size>
        <width>1021</width>
        <height>521</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1021</width>
        <height>521</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/image/toum.png);</string>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string>学号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>性别</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>班级</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>专业</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>签到框</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>加减分项</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;
border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>开始顺序点名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;
border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>暂停点名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;
border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>开始随机点名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;
border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>取消本次点名</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>60</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>80</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>60</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string>点名结果：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;</string>
   </property>
   <property name="text">
    <string>学 号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;</string>
   </property>
   <property name="text">
    <string>姓 名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;</string>
   </property>
   <property name="text">
    <string>性 别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;</string>
   </property>
   <property name="text">
    <string>班 级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;</string>
   </property>
   <property name="text">
    <string>专 业</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;</string>
   </property>
   <property name="text">
    <string>到 课</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;</string>
   </property>
   <property name="text">
    <string>备 注 信 息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;造字工房悦圆（非商用）常规体&quot;;</string>
   </property>
   <property name="text">
    <string>点名进度：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
